### DISCUSSION TITLE
Lists Accessibility

### DISCUSSION THREAD

**Discussion by Ro-Matos** [Telefonica/mistica-design#1630] [Proposals]
### Goal

Atribute semantics to Lists component to implement accessibility corretly.

1. Currently the List structure uses only `div`. Is necessary use `<ul>` and `<li>` to have semantic HTML. 
(Already reported in issue #1489)

### Context

In our accessibility specs developed by the design teams of APP Vivo, this cases have frequently documented. But the developers are not able to assign the all properties in the code.

Is possible to use "role=list" and "role=listItem" but this use don't atribute if the List is ordered or unordered. Another point is that more practical to devs if the semantic comes by default in the component.

### Impact

Improvements and compliance with accessibility guidelines.

### Discussion Questions

None

**Comment by yceballost** [comment]
I'm unclear about the purpose of this discussion. If the issue has already been identified and included for resolution, why initiate a separate discussion? 🤔

Please feel free to contribute directly to issue #1489 if there's something specific you'd like to emphasize or ensure is considered during its resolution

**Reply by Ro-Matos** [reply]
I created the discussion because in another component there was a similar situation (buttons), your guidance was to create a discussion to prioritize it (as a OB request):

![Captura de Tela 2024-02-02 às 09 07 31](https://github.com/Telefonica/mistica-design/assets/140208469/94c547b9-c3a8-42a1-adf3-0ef1af2aea07)

And this issue was despriorized
![Captura de Tela 2024-02-02 às 10 40 11](https://github.com/Telefonica/mistica-design/assets/140208469/913dd445-307b-468c-a884-6f5e1b0086d3)

**Reply by yceballost** [reply]
Yes, but the discussions are to discuss whether or not to include it in Mística.

Once a discussion is finalised an issue is created which means that it is a task to be worked on in Mística because it is something that will be included.

If there is something that you want to prioritise from the backlog of [issues](https://github.com/Telefonica/mistica-design/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc), the best thing to do is to create a ticket in the OBVIVO jira to prioritise it among all the others.

**Reply by yceballost** [reply]
the de-prioritisation is because we work with OKRs every quarter, and the accessibility project has moved down a lot on the Q1 target list (because we have other projects with higher priority).

I don't know if you can see this link, but here are listed the OKRs of my team. (I can show you in private if you want)
https://confluence.tid.es/display/CTO/2024-Q1+Team+Objectives%2C+Initiatives+and+KPIs#id-2024Q1TeamObjectives,InitiativesandKPIs-DesignCore

The first okr is to work on the components that you have asked for from Vivo, which are these (sorted by priority):
![image](https://github.com/Telefonica/mistica-design/assets/6722153/69c75e41-f486-4938-85dd-26031fe361aa)

Accessibility project is the 6!
![image](https://github.com/Telefonica/mistica-design/assets/6722153/e3f70998-e011-4bbb-8e5b-3d8c3f0a40b3)

When we finish with Vivo component list requests for Q1, we can work on more component that are listed in [OBVIVO Jira](https://jira.tid.es/secure/RapidBoard.jspa?rapidView=4402&projectKey=OBVIVO) but here your team will be able to prioritise whether to continue creating components or to prioritise improving accessibility.

I hope I have explained myself as well as possible!

**Reply by Ro-Matos** [reply]
My Jira access is still pending. I'll forward it to someone with access to include this accessibility issue as an OB prioritization.
Sorry and thanks Yayo!
